code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -354,6 +354,8 @@
 				if(HAS_TRAIT(src, TRAIT_DUMB))
 					msg += "[t_He] [t_has] a stupid expression on [t_his] face.\n"
 		if(getorgan(/obj/item/organ/brain))
+			if(ai_controller?.ai_status == AI_STATUS_ON)
+				msg += "<span class='deadsay'>[t_He] do not seem to be themselves.</span>\n"
```

Comment:
```suggestion
				msg += "<span class='deadsay'>[t_He] do[t_es]n't appear to be [t_him]self.</span>\n"
```
You'll also need to add `var/t_es = p_es()` at like line 9 with the other pronoun handlers, but I can't figure out how to suggest on a line you didn't touch. This will give `He doesn't appear to be himself/They don't appear to be themself` which is more grammatically correct. Message me on discord, I can write a quick pronoun handler for the 'f/ves' at the end of the message